python-examples/README.md

```diff
@@ -4,8 +4,254 @@ Termhub in 5 Minutes: python Tutorial
 ===================================================
 
 This tutorial shows how to use Python programming language to access content from the TermHub Terminology API.
+All the operations describes here perform a GET request.
 
 Prerequisites
 -------------
 
-COMING SOON!
+* Python 3.7+
+
+* All libraries in 'requirements.txt' must be installed.
+    * Run the command 'pip install -r requirements.txt' in a console window to check these libraries and install any
+      that are not already installed.
+    * If pip itself is not installed, run the command 'curl <https://bootstrap.pypa.io/get-pip.py> -o get-pip.py; python
+      get-pip.py' to install it.
+
+The various scripts make use of the `python-examples/config.ini` file to load necessary information that is uniform
+across all tests.
+
+When running the tests, make sure you are on the root folder.
```

Comment:
root folder?  maybe you should say the python_examples folder?

Reply:
Removed since python users will know how to run test